internal/server/kong/ws/server.go

```diff
@@ -119,6 +122,12 @@ func (h websocketHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		loggerWithNode.Error("failed to upgrade websocket connection", zap.Error(err))
 		return
 	}
+	// Tracks how long it takes to establish the connection between the control plane and the data plane.
+	metrics.Histogram("data_plane_connect_duration_seconds",
```

Comment:
```suggestion
	metrics.Histogram("wrpc_connection_setup_duration_seconds",
```

Once again, do not prefix with DP or CP, those qualifiers are not necessary.

Reply:
This code isn't just limited to wRPC, and given that we have an existing metric named `data_plane_ping_total`, this is following current conventions, right?

Reply:
It makes sense to keep these void of qualifiers. I know @omegabytes you have already committed the change to `wrpc_connection_setup_duration_seconds` but as @tjasko pointed out it does make sense to not include that qualifier either.

`connection_setup_duration_seconds` seems appropriate.